### PR TITLE
874842 - UG content added for Maui TreeMap control

### PR DESCRIPTION
**Pull request by Shalini-Ashokan**
### Description
Added UG content for MAUI Tree map control.

### CODE REVIEW

File: MAUI/TreeMap/events.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+---
+
+# Events in the .NET MAUI TreeMap (SfTreeMap)
+
+## SelectionChanged
+
+The `SelectionChanged` event occurs when there's a change in the selection within a tree map item.
+Below is a list of the arguments:
+
+* `Sender`: This contains the `SfTreeMap` object.
```
Comment (karthikraja-arumugam) [inline MAUI/TreeMap/events.md:17]: `Sender`: This contains the `SfTreeMap` object.

No need to add this

File: MAUI/TreeMap/events.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+{
+    public string Country { get; set; }
+    public string Continent { get; set; }
+    public int Population { get; set; }
+}
+
+{% endhighlight %}
+{% highlight c# tabtitle="PopulationViewModel.cs" %}
+
+/// <summary>
```
Comment (karthikraja-arumugam) [inline MAUI/TreeMap/events.md:82]: remove all summaries

File: MAUI/TreeMap/events.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+    </treemap:SfTreeMap.LeafItemBrushSettings>
+</treemap:SfTreeMap>
+
+{% endhighlight %}
+
+{% highlight C# hl_lines="1" %}
+
+this.treeMap.SelectionChanged += this.OnTreeMapSelectionChanged;
+
+ /// <summary>
```
Comment (karthikraja-arumugam) [inline MAUI/TreeMap/events.md:52]: remove summary

File: MAUI/TreeMap/events.md [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+# Events in the .NET MAUI TreeMap (SfTreeMap)
+
+## SelectionChanged
+
+The `SelectionChanged` event occurs when there's a change in the selection within a tree map item.
+Below is a list of the arguments:
+
+* `Sender`: This contains the `SfTreeMap` object.
+
+* `SelectionChanged`: This event occurs when a selection change action is performed on an element available in the `TreeMapSelectionChangedEventArgs` object.
```
Comment (karthikraja-arumugam) [inline MAUI/TreeMap/events.md:19]: We can get the below details from the `SelectionChanged` event.

    * `OldItems`: Returns the previously selected item.
    * `NewItems`: Returns the newly selected item.
    * `Category`: Returns the hierarchical path of the selected item within its groups.
    * `IsSelected`: Returns the items that are selected or deselected.
    * `GroupLevel`: Returns the level of grouping for the selected items.

File: MAUI/TreeMap/accessibility.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+<tr>
+<td>
+Shift + Tab<br/><br/></td><td>
+Moves selection to the previous item on the left side of the TreeMap.
+<br/><br/></td></tr>
+</table>
+
+N> 
+
+* When `SelectionMode` is set to `Single,` each key press clears the previous focus and selects the new item.
```
Comment (karthikraja-arumugam) [inline MAUI/TreeMap/accessibility.md:41]: Points will not be included in note, add separate note

File: MAUI/TreeMap/layouts.md [hunk 6]
```diff
@@ -0,0 +1,386 @@
...
+treeMap.LayoutType = LayoutType.Squarified;
+treeMap.PrimaryValuePath = "Population";
+treeMap.LeafItemSettings = new TreeMapLeafItemSettings() { LabelPath = "Country" };
+treeMap.LeafItemBrushSettings = new TreeMapUniformBrushSettings() { Brush = new SolidColorBrush(Color.FromArgb("#D21243")) };
+this.Content = treeMap;
+
+{% endhighlight %}
+{% highlight C# tabtitle="PopulationDetails.cs" %}
+
+/// <summary>
```
Comment (karthikraja-arumugam) [inline MAUI/TreeMap/layouts.md:53]: remove summaries in all examples

File: MAUI/TreeMap/layouts.md [hunk 7]
```diff
@@ -0,0 +1,386 @@
...
+        set;
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+![slice-and-dice-horizontal-layout-type-in-maui-treemap](images/layouts/slice-and-dice-horizontal-layout-type-in-maui-treemap.jpeg) {:width="313" height="444" .lazy .shadow-effect}
+
+## SliceAndDiceVertical
```
Comment (karthikraja-arumugam) [inline MAUI/TreeMap/layouts.md:294]: maintain the same naming
Slice and dice horizontal

File: MAUI/TreeMap/events.md (hunk 1 above)
Comment (Shalini-Ashokan) [reply MAUI/TreeMap/events.md]: Removed

File: MAUI/TreeMap/events.md (hunk 2 above)
Comment (Shalini-Ashokan) [reply MAUI/TreeMap/events.md]: Removed all summaries

File: MAUI/TreeMap/events.md (hunk 3 above)
Comment (Shalini-Ashokan) [reply MAUI/TreeMap/events.md]: removed

File: MAUI/TreeMap/events.md (hunk 4 above)
Comment (Shalini-Ashokan) [reply MAUI/TreeMap/events.md]: Updated as you mentioned

File: MAUI/TreeMap/accessibility.md (hunk 5 above)
Comment (Shalini-Ashokan) [reply MAUI/TreeMap/accessibility.md]: Added a note like paragraph

File: MAUI/TreeMap/layouts.md (hunk 6 above)
Comment (Shalini-Ashokan) [reply MAUI/TreeMap/layouts.md]: Removed summaries in all examples

File: MAUI/TreeMap/layouts.md (hunk 7 above)
Comment (Shalini-Ashokan) [reply MAUI/TreeMap/layouts.md]: Changed a title

**Review by M-A32 — CHANGES_REQUESTED** [review]
I have completed the review.

File: MAUI/TreeMap/accessibility.md [hunk 8]
```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title: Accessibility in .NET MAUI TreeMap (STreeMap) control | Syncfusion
```
[the remark this answers is not in the document]
Comment (Shalini-Ashokan) [reply MAUI/TreeMap/accessibility.md]: Updated the title

File: MAUI/TreeMap/accessibility.md [hunk 9]
```diff
@@ -0,0 +1,43 @@
...
+
+The TreeMap is designed to effectively work with its elements, providing voice descriptions of the treemap items.
+
+## Keyboard shortcuts in .NET MAUI TreeMap (SfTreeMap)
+
+The `Syncfusion .NET MAUI TreeMap` supports keyboard shortcuts for user interaction. 
+
+N> This feature is only applicable for Windows and macOS platforms.
+
+Below is a table outlining the various shortcuts and their associated functions:
```
[the remark this answers is not in the document]
Comment (Shalini-Ashokan) [reply MAUI/TreeMap/accessibility.md]: Changed the content

File: MAUI/TreeMap/events.md [hunk 10]
```diff
@@ -0,0 +1,120 @@
+---
+layout: post
+title: Events in .NET MAUI TreeMap (SfTreemap) control | Syncfusion
```
[the remark this answers is not in the document]
Comment (Shalini-Ashokan) [reply MAUI/TreeMap/events.md]: Updated the title

File: MAUI/TreeMap/events.md [hunk 11]
```diff
@@ -0,0 +1,120 @@
...
+platform: maui
+control: TreeMap (SfTreeMap)
+documentation: ug
+---
+
+# Events in the .NET MAUI TreeMap (SfTreeMap)
+
+## SelectionChanged
+
+The `SelectionChanged` event occurs when there's a change in the selection within a tree map item.
```
[the remark this answers is not in the document]
Comment (Shalini-Ashokan) [reply MAUI/TreeMap/events.md]: Updated the content

File: MAUI/TreeMap/layouts.md [hunk 12]
```diff
@@ -0,0 +1,386 @@
+---
+layout: post
+title: Layout types in .NET MAUI TreeMap control | Syncfusion
```
[the remark this answers is not in the document]
Comment (Shalini-Ashokan) [reply MAUI/TreeMap/layouts.md]: Updated the title

File: MAUI/TreeMap/layouts.md [hunk 13]
```diff
@@ -0,0 +1,386 @@
...
+description: Learn here all about adding different layouts of the Syncfusion .NET MAUI TreeMap (STreeMap) control and more.
+platform: maui
+control: TreeMap (SfTreeMap)
+documentation: ug
+---
+ 
+# Layouts in .NET MAUI TreeMap (SfTreeMap)
+
+The Tree Map control provides multiple layout types for organizing hierarchical data effectively. You can utilize the `LayoutType` property within the SfTreeMap to switch between four distinct layouts: `Squarified,` `SliceAndDiceAuto,` `SliceAndDiceHorizontal,` and `SliceAndDiceVertical.`
```
[the remark this answers is not in the document]
Comment (Shalini-Ashokan) [reply MAUI/TreeMap/layouts.md]: Updated the content as you mentioned

File: MAUI/TreeMap/layouts.md [hunk 14]
```diff
@@ -0,0 +1,386 @@
...
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+![slice-and-dice-auto-layout-type-in-maui-treemap](images/layouts/slice-and-dice-auto-layout-type-in-maui-treemap.jpeg) {:width="313" height="444" .lazy .shadow-effect}
+
+## Slice and dice horizontal
+
+The `SliceAndDiceHorizontal` layout arranges data into horizontal rectangles with a high aspect ratio and displays them sorted horizontally. This layout will start to arrange each rectangle in a horizontal direction and the size of the rectangle will be calculated based on the `PrimaryValuePath` property value.
```
[the remark this answers is not in the document]
Comment (Shalini-Ashokan) [reply MAUI/TreeMap/layouts.md]: Updated the content as you mentioned

File: MAUI/TreeMap/layouts.md [hunk 15]
```diff
@@ -0,0 +1,386 @@
...
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+![slice-and-dice-horizontal-layout-type-in-maui-treemap](images/layouts/slice-and-dice-horizontal-layout-type-in-maui-treemap.jpeg) {:width="313" height="444" .lazy .shadow-effect}
+
+## SliceAndDiceVertical
+
+The `SliceAndDiceVertical` layout organizes data into vertical rectangles with a high aspect ratio and displays them sorted vertically. This layout will start to arrange each rectangle in a vertical direction and the size of the rectangle will be calculated based on the `PrimaryValuePath` property value.
```
[the remark this answers is not in the document]
Comment (Shalini-Ashokan) [reply MAUI/TreeMap/layouts.md]: Updated the content as you mentioned

File: MAUI/TreeMap/legend.md [hunk 16]
```diff
@@ -0,0 +1,888 @@
+---
+layout: post
+title: Legend in .NET MAUI TreeMap control | Syncfusion
```
[the remark this answers is not in the document]
Comment (Shalini-Ashokan) [reply MAUI/TreeMap/legend.md]: Updated the title

**Review by karthikraja-arumugam — APPROVED** [review]
This is fine

**Pull request merged**